scheduler/wrapper.go

```diff
@@ -53,13 +53,24 @@ func PlanDistro(ctx context.Context, conf Configuration, s *evergreen.Settings)
 			if err != nil {
 				grip.Error(err)
 			}
+			err = model.ClearTaskQueue(distro.Id)
```

Comment:
If you call `ClearTaskQueue`, is there a reason to also call `RemoveTaskQueues` first?